sampleExtension/statistics/trackerStorage.js

```diff
@@ -0,0 +1,123 @@
+
+var TrackerStorage = new function() {
+    var self = this;
+
+    this.init = function(){
+        // Create new dummydata.
+        self.setDayStatisticsList([]);
+        self.loadDummyData(10);
+
+        // Make sure there is a record for the current day by calling this method.
+        self.incrementDayStat(0);
+    };
+
+
+    // Loads dummy data in the previous day list for demonstration purposes.
+    this.loadDummyData = function(amount){
+        var dummyList = [];
+        for(var i = amount; i > 0; i--){
+            var date = new Date();
+            date = new Date(date.setDate(date.getDate()-i));
+            dummyList.push({'date': ((date.getDate())+"/"+(date.getMonth()+1)+"/"+date.getFullYear()), 'timespent': Math.floor((Math.random()*100)+1)});
```

Comment:
The value associated with the key `date` is incomprehensible, move to a function.